### PR TITLE
F/ref bank

### PR DESCRIPTION
**Pull request by johanlbatu**
I have been created module ref-banks **[ update , delete , create ]**

### CODE REVIEW

File: resources/views/vendor/adminlte/ref-banks/edit.blade.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            <hr>
+
+            <form action="{{ url('ref-banks/'.$refBanks->id) }}" method="post" data-toggle="validator" role="form" enctype="multipart/form-data">
+                <input type="hidden" name="_method" value="PUT">
+                {{ csrf_field() }}
+                <div class="form-group">
+                    <label>Name</label>
+                    <label>Name</label>
+                    <input type="text" name="account_vendor" class="form-control" value="{{$refBanks->account_vendor}}" data-error="Please enter name" required> <br>
+                    <input type="text" name="account_number" class="form-control" value="{{$refBanks->account_number}}" data-error="Please enter name" required>
```
Comment (sjuliper7) [inline resources/views/vendor/adminlte/ref-banks/edit.blade.php]: please add form group for input account number **johan**

Comment (johanlbatu) [reply resources/views/vendor/adminlte/ref-banks/edit.blade.php]: Ok..

File: app/RefBank.php [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class RefBank extends Model
+{
+    protected $fillable = [
+        'account_vendor', 'account_number', 'id',
```
Comment (sjuliper7) [inline app/RefBank.php]: please remove id form fillable "because id is auto increments so id is auto fillable"

Comment (johanlbatu) [reply app/RefBank.php]: done..

**Pull request merged**